### PR TITLE
feat: inject tegg ctx to mocha runner

### PR DESCRIPTION
**Pull request by killagu**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -465,6 +466,13 @@ describe('test app', () => {
       .expect(/bar/);
   });
 });
+
+describe('test ctx', () => {
+  it('can use ctx', async function() {
+    const res = await this.ctx.service.foo();
```
Comment (fengmk2) [inline README.md:472]: 箭头函数也支持吧？

```js
it('can use ctx', async () => {
  const res = await this.ctx.service.foo();
});
```

**Review by fengmk2 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (killagu) [reply README.md:472]: 箭头函数不行的。箭头函数的 this 不能变。

**Review by fengmk2 — APPROVED** [review]

**Comment by fengmk2** [comment]
自动发布出问题了，我先手动发一个。

**Comment by fengmk2** [comment]
5.9.0

**Pull request merged**